torchvision/prototype/transforms/_augment.py

```diff
@@ -14,7 +14,7 @@
 from torchvision.transforms.v2.utils import has_any, is_simple_tensor, query_size
 
 
-class _BaseMixupCutmix(_RandomApplyTransform):
+class _BaseMixUpCutMix(_RandomApplyTransform):
```

Comment:
Can't we delete them from prototype now that we have a proper implementation in v2? If you do, note that there are tests for this in `test/test_prototype_transforms.py`

Reply:
will do in another PR